### ISSUE TITLE
parseCidr with 32 is not valid

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#2336]
**Description of the issue**

When using a `/32` IP address with the `parseCidr` and `cidrSubnet` function the CIDR is reported as invalid.

This was reported in #2279.

**Error output**

```text
An error occurred evaluating expression '[parseCidr(parameters('AllowedPublicIp').CIDR[copyIndex()]).firstUsable]' line #.
The specified CIDR '108.223.209.248/32' is not valid.
```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.28.0**

**Comment by tstooke** [comment]
@BernieWhite Thank you for being so quick with these!
